src/inquirer/questions.py

```diff
@@ -32,8 +33,23 @@ def __ne__(self, other):
 class Question:
     kind = "base question"
 
-    def __init__(self, name, message="", choices=None, default=None, ignore=False, validate=True, show_default=False):
+    def __init__(
+        self,
+        name,
+        message="",
+        choices=None,
+        default=None,
+        ignore=False,
+        validate=True,
+        show_default=False,
+        other=False,
+    ):
+        if other:
+            choices = choices or []
+            choices.append(GLOBAL_OTHER_CHOICE)
+
         self.name = name
+        self._other = other
```

Comment:
I would find it more logical, if they would match the order in which they are passed to the function, so maybe move it down?